src/app/execution/txLogger.js

```diff
@@ -44,36 +42,35 @@ var css = csjs`
     float: left;
   }
   .succeeded {
-    color: ${styles.terminal.icon_Color_Log_Succeed};
+    color: var(--success);
   }
   .failed {
-    color: ${styles.terminal.icon_Color_Log_Failed};
+    color: var(--danger);
   }
   .notavailable {
   }
   .call {
     font-size: 7px;
-    background-color: ${styles.terminal.icon_BackgroundColor_Log_Call};
     border-radius: 50%;
     min-width: 20px;
     min-height: 20px;
     display: flex;
     justify-content: center;
     align-items: center;
-    color: ${styles.terminal.icon_Color_Log_Call};
+    color: var(--primary);
     text-transform: uppercase;
     font-weight: bold;
   }
   .txItem {
-    color: ${styles.terminal.text_Primary};
+    color: var(--primary);
     margin-right: 5px;
     float: left;
   }
   .txItemTitle {
     font-weight: bold;
   }
   .tx {
-    color: ${styles.terminal.text_Title_TransactionLog};
+    color: var(--primary);
```

Comment:
This is comment about the hover color - its hard to tell that this is a link by the hover color.  Although I don't see the hover color here in this css.

Reply:
don't see the hover declaration here..